### DISCUSSION TITLE
Need help about inheritance mapping polymorphic relationships

### DISCUSSION THREAD

**Discussion by bocanhcam** [doctrine/doctrine2#10333] [Support Questions]
![Screenshot from 2022-12-20 17-04-46](https://user-images.githubusercontent.com/35592955/208804057-25f2139e-7b75-4a7f-805c-29962677f4cf.png)

I have 3 tables like the above.
I want to make the relationship between column Article with Customer and Member using ```creator_id``` and ```creator_type```.
I'm struggling with how to create relationships for these table entities.
I already use ```@ORM\DiscriminatorColumn(name="discriminator_type", type="string", length=255)``` to define the type of each entity so I don't think of another way how to do it.
Can anyone give me a sample code?
Thanks a lot!!!
By the way, I use Symfony 5

**Comment by mpdude** [comment]
I see what you’re trying to do, but relational databases don’t work that way. You cannot have the creator_id reference either a member or a customer with a foreign key, depending on the creator_type value.

(Yes, you can do this without foreign keys, but again, that’s not how relational DBs are designed to work, and it won’t work with the ORM that way.)

**Reply by mpdude** [reply]
Oh, I think you're asking for this: https://github.com/doctrine/orm/discussions/9928
